lib/evmone/instructions.cpp

```diff
@@ -1066,16 +1018,12 @@ void op_create(execution_state& state, instr_argument arg) noexcept
 
     msg.sender = state.msg->destination;
     msg.depth = state.msg->depth + 1;
-    intx::be::store(msg.value.bytes, endowment);
+    msg.value = store<evmc::uint256be>(endowment);
```

Comment:
In other places you do it like `store(msg.value.bytes, value);`
I like this assignment form better, maybe change other places for consistency?

Reply:
Fixed.